### ISSUE TITLE
Is %%freegchord implemented? Or, is there a way not to replace b with ♭?

### ISSUE THREAD

**Issue by db-ryan** [paulrosen/abcjs#264]
The font I'm using doesn't include ♭.

![image](https://user-images.githubusercontent.com/1309807/67164529-90434280-f349-11e9-8e73-833cb58e44c9.png)

If this isn't implemented, if you point me in the file/direction of where it 'could' be implemented, I'd be happy to do a PR.

Thanks!

**Comment by paulrosen** [comment]
What I did is just implement the symbols I needed in my own font. Here's an example:
![example-font](https://user-images.githubusercontent.com/69818/67208370-66d6f500-f3e3-11e9-9c36-7926c1552fc0.png)

Anyway, I can tell you the tools I cobbled together to do that (I'm not a font expert so I'm sure I didn't do it the easy way.)

But your question is about freegchord - I wasn't aware of that option. I see this definition:

freegchord boolean
    Prevent the characters `#', `b' and `=' to be displayed as the sharp sign, the flat sign, and the natural sign, respectively, in guitar chords. When this flag is set, the display of the accidentals may be forced by escaping the characters (`\#', `\b' and `\=') (default: 0) 

That doesn't seem very hard to implement. `src/parse/abc_parse_directive.js` is were to define it. It goes into the multiline variables. then look for `letter_to_chord` in `src/parse/abc_parse.js` and you'll see the substitution.
